source/core/Events.cpp

```diff
@@ -871,9 +871,9 @@ VIREO_FUNCTION_SIGNATUREV(RegisterForEvents, RegisterForEventsParamBlock)
     }
     RefNum erRefNum = eventRegRefnumPtr->GetRefNum();
     if (erRefNum && EventRegistrationRefNumManager::RefNumStorage().GetRefNumData(erRefNum, &regInfo) != kNIError_Success) {
-        // event reg. ref passed, but invalid
-        if (errPtr && !errPtr->status)
-            errPtr->SetErrorAndAppendCallChain(true, kEventArgErr, "RegisterForEvents");
+        // event reg. ref passed, but invalid. However, this is not user error.
```

Comment:
Can we tweak this so the side effect is not in the conditional since we aren't doing anything in the conditional anyway?
```cpp
if (erRefNum) {
    EventRegistrationRefNumManager::RefNumStorage().GetRefNumData(erRefNum, &regInfo);
    // comment about how it can still be invalid
}
```